src/databricks/labs/ucx/framework/tasks.py

```diff
@@ -53,7 +55,11 @@ def wrapper(*args, **kwargs):
             msg = f"Task {func.__name__} must have documentation"
             raise SyntaxError(msg)
 
+        global _TASK_ID_GENERATOR
```

Comment:
@nfx The linter does not like this:
```
src/databricks/labs/ucx/framework/tasks.py:58:9: PLW0603 Using the global statement to update `_TASK_ID_GENERATOR` is discouraged
```
Is there a better way?

Reply:
You can len(_TASKS)

Reply:
no additional counter to track, right?